config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -388,3 +388,36 @@ periodics:
     testgrid-tab-name: kubelet-serial-gce-e2e-graceful-node-shutdown
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
     description: Executes graceful node shutdown e2e node tests
+
+- name: ci-kubernetes-node-kubelet-node-swap
+  interval: 4h
```

Comment:
We don't _need_ a periodic test until the code merges (since we won't be able to enable the feature flag yet), but I'm fine with enabling this with the kubelet flag + swap enabled for now. It should be a no-op.

Reply:
Ack. I'll just remove this section for now then. Will add back when code is merged.